standalone-metastore/metastore-server/pom.xml

```diff
@@ -573,6 +573,9 @@
             <java.io.tmpdir>${test.tmp.dir}</java.io.tmpdir>
             <test.tmp.dir>${test.tmp.dir}</test.tmp.dir>
             <hive.in.test>true</hive.in.test>
+            <derby.version>${derby.version}</derby.version>
+            <derby.stream.error.file>${test.tmp.dir}/derby.log</derby.stream.error.file>
+            <javax.jdo.option.ConnectionURL>jdbc:derby:memory:${test.tmp.dir}/junit_metastore_db;create=true</javax.jdo.option.ConnectionURL>
```

Comment:
This is a `MetastoreConf` property, it doesn't feel right to put it here. Also, I am not sure if every test should use an Derby database and specifically in-memory. Why is this change needed?

Reply:
The following line is needed, and the derby.log file is generated inside the package directory for `ITestDerby` test.
```
<derby.stream.error.file>${test.tmp.dir}/derby.log</derby.stream.error.file>
```

Removed the `javax.jdo.option.ConnectionURL` property